src/Backend/Synapse/Importer.php

```diff
@@ -97,6 +99,18 @@ public function importTable(
                 );
             }
             $this->importState->setImportedColumns($source->getColumnsNames());
+        } catch (DBALException $e) {
+            if (strpos($e->getMessage(), '[SQL Server]Bulk load') !== false) {
```

Comment:
Nema to este nejaky spesl error code alebo tak ?

Reply:
Error code je  `0` :D budu rád když sa tá hláška nezmění :D

Reply:
hmm tak asi nevymyslime nic lepse :/